cassandra-analytics-cdc-codec/src/main/resources/cdc_bytes.avsc

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
```

Comment:
Interesting. I am not aware that avro spec allows comments like this. https://avro.apache.org/docs/++version++/specification/
Does it work?

Reply:
Good catch!! I miss read that it was supported on a different format for defining avro called IDL.
https://avro.apache.org/docs/1.11.1/idl-language/#comments

So, no, it won't work. I guess we'll have to add exceptions for the licensing checks on these Avro schemas?

Reply:
OK. It actually works. I also verified locally that the schema with the comment can be correctly parse.